content/main.tex

```diff
@@ -1,5 +1,7 @@
 \documentclass[american]{article}
-
+\usepackage{titlesec}
+\usepackage{graphicx}
+\graphicspath{ {images/} }
```

Comment:
Change this path to reflect the image directory path from the perspective of the out directory I believe. Basically whichever path works for the Makefile.